notebooks/tfx_pipelines/walkthrough/solutions/tfx_walkthrough_vertex.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>Remove last line or update to Vertex Pipelines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>

Reply:
<p>Done!</p>

Reply:
<p>Thanks for the review!</p>